bootstrap/command/test

```diff
@@ -33,3 +33,16 @@ if is_already_formatted; then
 else
   echo "You should run |bacardi format| command first."
 fi
+
+#if is_windows_platform; then
```

Comment:
This project is not Chromium but we are following Chromium style and policy if possible.
In Chromium, we try to avoid unused code or unused comments if possible.
Once the code is merged, it requires some reasons to remove them.
So, IMHO, it's better to remove these comment lines. WDYT?

Reply:
Ah... Thanks to you, I came to know Chromium Style. I will remove this comments!